python/semantic_kernel/connectors/memory/azure_cognitive_search/azure_cognitive_search_memory_store.py

```diff
@@ -105,7 +105,7 @@ async def create_collection(
         else:
             vector_search = VectorSearch(
```

Comment:
More precisely that's `VectorSearch` class, we need to change the current parameters [VectorSearch Class](https://azuresdkdocs.blob.core.windows.net/$web/python/azure-search-documents/11.6.0b1/azure.search.documents.indexes.models.html#azure.search.documents.indexes.models.VectorSearch)